task/06-conditions-n-loops-tasks.js

```diff
@@ -296,7 +349,35 @@ function getDigitalRoot(num) {
  *   '{[(<{[]}>)]}' = true 
  */
 function isBracketsBalanced(str) {
-    throw new Error('Not implemented');
+    const brackets = new Map([
```

Comment:
Map is used when you need to get value by a key. Here you use keys to check for opening bracket and values to check for closing brackets. Why don't just use 2 arrays and search with indexof?